examples/ProductCollectionPlanet.json

```diff
@@ -2,103 +2,201 @@
     "products": [
         {
             "type": "Product",
-            "stat_version": "stat_version",
-            "stat_extensions": [],
-            "id": "PL-QA:Assured Tasking",
-            "title": "Assured Tasking",
-            "description": "An assured capture at a specific time and location.",
-            "keywords": ["EO", "color"],
+            "conformsTo": [
+                "https://api.statspec.org/geojson#point",
+                "https://api.statspec.org/geojson#linestring"
+            ],
+
+            "id": "PL-123456:Assured Tasking",
+            "title": "SkySat Assured Tasking",
+            "description": "An assured SkySat capture at a specific time and location.",
+            "keywords": [
+                "EO",
+                "color",
+                "assured"
+            ],
             "license": "Planet",
             "providers": {
-                "name": "planet",
-                "description": "planet description",
+                "name": "Planet",
+                "description": "Planet Labs, Inc.",
                 "roles": ["producer"],
-                "url": "planet link"
+                "url": "https://planet.com/"
             },
```

Comment:
providers should be the list of object. Migrating `{...}` -> `[{...}]`

Reply:
thx!